src/autocomplete/typeahead.js

```diff
@@ -85,16 +85,33 @@ function Typeahead(o) {
     .onSync('queryChanged', this._onQueryChanged, this)
     .onSync('whitespaceChanged', this._onWhitespaceChanged, this);
 
+  this._bindKeyboardShortcuts(o, $input);
+
   this._setLanguageDirection();
 }
 
 // instance methods
 // ----------------
 
 _.mixin(Typeahead.prototype, {
-
   // ### private
 
+  _bindKeyboardShortcuts: function (o, $input) {
+    DOM.element(document).keydown(function(e) {
+        if (! o.keyboardShortcuts) {
+          return;
+        }
+
+        var which = e.which || e.keyCode;
+        if (false === $input.is(':focus') && o.keyboardShortcuts.indexOf(which) !== -1) {
```

Comment:
That won't be enough; if you're typing something in another input you'll trigger that because the searchbar is not focused.